### ISSUE TITLE
Twitter and Facebook Posts - Sidebars

### ISSUE THREAD

**Issue by spazm6666** [jaredhaight/lgod#3]
Sidebar posts do not show on Twitter and Facebook. By design? Oversight? Thanks.

**Comment by jaredhaight** [comment]
That was by design.. sort of. The RSS feed was designed to only pull standard and featured articles. Since sidebar articles don't have a view page to link to, they were omitted. 

Since Facebook and Twitter updates are currently pulled by polling RSS, they don't show sidebar articles.  I could see the value in posting those to facebook as status updates though. Twitter probably wouldn't work because of the char limit. 

What I can do is put together a special, unpublished RSS feed for sidebars and have IFTTT pull that and post them to the Facebook page. Before I do that though, I'd want to make sure that that behavior was kosher with Justin and Colin.

**Comment by spazm6666** [comment]
That was the exact response I had formulated in my head. Were you also not wearing pants when you wrote it? Like in my head?

**Comment by jaredhaight** [comment]
@cprincipe and @djhaighted - Thoughts on how you want this handled?

**Comment by djhaighted** [comment]
The direct posting to Facebook is great idea. I have the goal to use the FB page more heavily this go around so having that content will help to fill out the page. Would placing the same text limit on sidebar articles that Twitter has help as well?  At the very least, this would guarantee the simplicity of the post as they are really meant to be just blurbs. That may even help to get them posted to twitter as is and not point back to LGOD. If Twitter is not an option at all, that is not a huge loss though.

**Comment by jaredhaight** [comment]
Yes, limiting the sidebar articles to 140 chars would allow them to be posted to twitter. Personally I think that would be too limiting, but I can make the site do what ever you want.

**Comment by djhaighted** [comment]
The Twitter posting of sidenotes is not too important to me.  @cprincipe? @spazm6666?
